### DISCUSSION TITLE
11.8.2023

### DISCUSSION THREAD

**Discussion by charlieforward9** [charlieforward9/ATHLEAT#49] [Team Meetings]
@rensaketh 
* Need to add a parameter to Database
   * Auth token, Refresh Token 
* #13

@charlieforward9 
* Work with Tico on shape of charts 
* Finish filter and service implementations
* #19
* #20 
* #22 
* #23 
* #25
* #26 
* #28 
* #29 

@PrestonGoren

Lambda Input of:
```
interface NutritionFormData { 
  date: string;
  time: string;
  food: string;
  quantity: number;
}
```
Output of EventJSON that Lambda stores to DB

* #7 

@ticogarcia 
* Finish all chart and filter views
* #21 
* #24 
* #27 
* #30

**Comment by charlieforward9** [comment]
@ticogarcia Double Axis Chart: https://www.chartjs.org/docs/latest/samples/line/multi-axis.html

**Comment by charlieforward9** [comment]
Sloppy Notes regarding Shape of Chart Data

![image](https://github.com/charlieforward9/ATHLEAT/assets/62311337/d607c370-a1c4-4c2d-b650-164f8fd17b09)

**Comment by rensaketh** [comment]
New Amplify Pull Command: amplify pull --appId d3s62klvptbnay --envName dev
